CWE-94-Python_darwin/Youlixiya_instance3dgs/3.py

```
@@ -0,0 +1,116 @@
+
+def qwen_template(prompt):
+    return f'Output the bounding box of the object that best matches the text description, not the bounding box of an irrelevant object. Please grounding <ref> {prompt} </ref>'
+
+def extract_box(text, w, h):
+    pattern = r'\((.*?)\)'
+    matches = re.findall(pattern, text)
+    box = []
+    for match in matches:
+        box += match.split(',')
+    for i in range(len(box)):
+        box[i] = eval(box[i])
```

Ruff (0.13.2)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)